clang/lib/AST/Expr.cpp

```diff
@@ -2237,6 +2237,21 @@ bool BinaryOperator::isNullPointerArithmeticExtension(ASTContext &Ctx,
   return true;
 }
 
+bool BinaryOperator::oneOfWraps(const ASTContext &Ctx) const {
+  llvm::SmallVector<Expr *, 2> Both = {getLHS(), getRHS()};
```

Comment:
I had trouble with your code snippet because `children()` gives `Stmt` iterators and I would have to add in a cast or two. I went with a simpler approach in a2f63982920f22d795c4971800bcc5cb55356570 (with QualType::hasWrapsAttr())